geopandas/tools/clip.py

```diff
@@ -110,10 +135,15 @@ def clip(gdf, mask, keep_geom_type=False):
             "'gdf' should be GeoDataFrame or GeoSeries, got {}".format(type(gdf))
         )
 
-    if not isinstance(mask, (GeoDataFrame, GeoSeries, Polygon, MultiPolygon)):
+    if not isinstance(mask, (GeoDataFrame, GeoSeries, Polygon, MultiPolygon, tuple)):
+        raise TypeError(
+            "'mask' should be GeoDataFrame, GeoSeries,"
+            f"(Multi)Polygon or 4 element tuple, got {type(mask)}"
```

Comment:
```suggestion
            f"(Multi)Polygon or a tuple, got {type(mask)}"
```
We  check the length of a tuple below so no need to mention it in the error.

Reply:
Done